pkg/bootstrap/manifests/klusterlet/bootstrap_secret.yaml

```diff
@@ -14,6 +14,19 @@ type: Opaque
 data:
   kubeconfig: "{{ $bootstrapkubeconfigsecret.KubeConfig }}"
 {{end}}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ .DefaultBootstrapKubeConfigSecret.Name }}-current-hub"
```

Comment:
maybe call it `original-hub`?

Reply:
The `original` imples that is the first, initial hub, but in our implementation, this piece of book-kubeconfig always points to the hub currentlly connectted with.